CMakeLists.txt

```diff
@@ -184,14 +184,20 @@ target_link_libraries(scylla PRIVATE
 # Force SHA1 build-id generation
 set(default_linker_flags "-Wl,--build-id=sha1")
 include(CheckLinkerFlag)
-foreach(linker "lld" "gold")
```

Comment:
I think we might want to preserve the previous behavior of using `lld` or `gold` if there is no user specified linker. AFAIK the default linker `GNU ld` is veeerrryyyy slow.

Reply:
@denesb sure. updated accordingly.